dev-python/terminado/terminado-0.8.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Why oh why?

Reply:
Because it's a simple copy and I have overlooked this. Will fix it.